scapy/layers/tls/session.py

```diff
@@ -1167,8 +1167,11 @@ def tcp_reassemble(cls, data, metadata, session):
             length = struct.unpack("!H", data[3:5])[0] + 5
             if len(data) >= length:
                 # get the underlayer as it is used to populate tls_session
-                underlayer = metadata["original"][TCP].copy()
-                underlayer.remove_payload()
+                if metadata.get("original", None) is None:
+                    underlayer = TCP()
+                else:
+                    underlayer = metadata["original"][TCP].copy()
+                    underlayer.remove_payload()
```

Comment:
This would be more correct:
```suggestion
                if "original" not in metadata:
                    return cls(data)
                underlayer = metadata["original"][TCP].copy()
                underlayer.remove_payload()
```